themes/default/content/blog/log4shell-response/index.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "Dealing with Log4Shell"
+date: 2021-12-13
+draft: false
+meta_desc: |
+    Pulumi does not use log4j and is not directly affected. However, if you want
+    to learn more about mitigation techniques as users of third-party systems,
+    read on.
+meta_image: meta.png
+authors:
+    - laura-santamaria
+tags:
+    - security
+---
+
+The recent security vulnerability CVE-2021-44228, or Log4Shell, that affects the
```

Comment:
i think we may want to figure out how to include the Pulumi is not directly affected bit in the first sentence of this

Reply:
and/or include it in the blog title

Reply:
Fair point. Fixing now.